js/notify/electron-notify-preload.js

```diff
@@ -88,13 +94,9 @@ function setContents(event, notificationObj) {
         if (notificationObj.color.match(whiteColorRegExp)) {
             logo.src = './assets/symphony-logo-black.png';
         } else {
-            let title = notiDoc.getElementById('title');
-            let pod = notiDoc.getElementById('pod');
-            let message = notiDoc.getElementById('message');
-
-            message.style.color = '#ffffff';
-            title.style.color = '#ffffff';
-            pod.style.color = notificationObj.color;
+            messageDoc.style.color = '#ffffff';
+            titleDoc.style.color = '#ffffff';
```

Comment:
What about in cases where we have a light theme?

Reply:
Other than white there are no light colours defined in the client app for the notifications.

We don't need to modify anything until they define colours lighter than these 
![colors](https://user-images.githubusercontent.com/13243259/33005062-2985d66a-cdea-11e7-9b71-9432e9d6935f.png)